cosmopolitan_countries/migrations/0001_initial.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('cosmopolitan_continents', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='CosmopolitanCountry',
+            fields=[
+                ('id', models.CharField(max_length=2, serialize=False, primary_key=True)),
+                ('name', models.CharField(max_length=200, verbose_name='ascii name', db_index=True)),
+                ('slug', models.CharField(max_length=200)),
+                ('population', models.IntegerField()),
+                ('code3', models.CharField(max_length=3)),
+                ('currency_short', models.CharField(max_length=3, null=True)),
```

Comment:
it makes things bit easier with initial import of data, but actually you are right, not so important to save it explicitly. I'll remove that.